disco_aws_automation/disco_metanetwork.py

```diff
@@ -11,6 +11,8 @@
     NetworkInterfaceCollection
 )
 from boto.exception import EC2ResponseError
+from boto.vpc import VPCConnection
+import boto3
```

Comment:
Only partially right now. It's used with NAT gateways because boto2 doesn't support it. We'll get rid of boto2 completely in the future.